doc/reactions.md

```diff
@@ -0,0 +1,152 @@
+[<- Back to Index](index.md)
+
+# Reactions
+
+Reactions allow SQL LRS to watch for patterns in submitted xAPI data and dynamically generate new statements in response.
+
+## Usage
```

Comment:
This section should include how to retrieve/query the statements generated by the reaction.

Reply:
Actually not only is direct query something we don't document, but it would be essentially impossible as they're just additional statements. They would need to put something unique in there, then query for that. Highly variable.